pdns/misc.cc

```diff
@@ -845,7 +845,6 @@ void addCMsgSrcAddr(struct msghdr* msgh, void* cmsgbuf, const ComboAddress* sour
     memset(pkt, 0, sizeof(*pkt));
     pkt->ipi6_addr = source->sin6.sin6_addr;
     pkt->ipi6_ifindex = itfIndex;
-    msgh->msg_controllen = cmsg->cmsg_len; // makes valgrind happy and is slightly better style
```

Comment:
Sounds good. I'll stick a comment in for when somebody finds it with valgrind again.